tests/util.py

```diff
@@ -357,6 +359,33 @@ def _perform_request(self, path, **kwargs):
 
         return self.client.open(path, **kwargs)
 
+    def sort_inner_lists(self, obj):
```

Comment:
This method doesn't actually refer to or use `self` anywhere — I'd say this should be a regular function instead. If you only use it within the other method, just having it as an internal helper function should suffice.